### PR TITLE
Kevin updates

### PR DESCRIPTION
**Pull request by FrankErrickson**

### CODE REVIEW

**Comment by FrankErrickson** [comment]
@kevinkuruc @davidanthoff @fdennig This is a pull request based off of Kevin's branch to update the final paper submission code. Two things I noticed that we need to sort out for the time being are:

(1) The cost-minimization vs. uniform price labeling (which Kevin updated in the code). We can't accept the pull request until this is sorted out.

(2) In the `src/helper_functions.jl` file, optimal emissions are saved as industrial emissions only. Minor issue, but should this also include the exogenous land-use emissions scenario?

If you spot anything else, please comment about it here.

File: src/run_analysis.jl [hunk 1]
```diff
@@ -136,28 +148,28 @@ end
 
 
 #------------------------------------------------------------------------------------------------------
-# Run FUND Cost-Minimization Optimization.
+# Run FUND Uniforcost_minimization Optimization.
```
Comment (FrankErrickson) [inline src/run_analysis.jl]: @kevinkuruc  Not sure what the optimization name should be here.

File: src/run_analysis.jl [hunk 2]
```diff
@@ -136,28 +148,28 @@ end
...
 #------------------------------------------------------------------------------------------------------
 if fund_cost_minimization == true
 
-    println("Starting FUND cost-minimization optimization...")
+    println("Starting FUND uniforcost_minimization optimization...")
```
Comment (FrankErrickson) [inline src/run_analysis.jl]: Same typo error here.

Comment (kevinkuruc) [reply src/run_analysis.jl]: Thanks, got them. Tried changing too much with a single cursor.

**Pull request merged**